docs/monitoring/monitoring.md

```diff
@@ -17,225 +18,195 @@ The following topics are covered in this section:
-
-* **Tables Count**</br>
-	Displays the total number of external tables present in the cluster. The tooltip displays the total number of external tables.
-
-* **Name**</br>
-	Displays the name of the external table.
-
-* **Provider**</br>
-	Displays the datastores provide used when creating the external table. For example, Parquet, CSV, JDBC etc.
+<a id="member"></a>
+### Members
+In the **Members** section,  you can view, in a tabular format, the details of each locator, server, and lead member within a cluster. The details are automatically refreshed after every 5 seconds.
+![Members](../Images/Dashboard-MembersList.png)
 
-* **Source**</br>
-	For Parquet and CSV format, the path of the data file used to create the external table is displayed. For JDBC, the name of the client driver is displayed.
+This table provides member details in the following columns:
 
-<a id="member-details"></a>
-## Member Details View
 
-The Member Details View offers the following capabilities and benefits:
+| Column | Description |
+|--------|--------
+|  **Status**     |  Displays the status of the members, which can be either [Running or Stopped](#statusofmembers).    |
+|   **Member**       |   Displays a brief description of the member. Click the link in the column to view detailed description of the members along with the [Member Logs](#memberlogs). In this description, you can find information such as the IP address of the host, the current working directory, and the Process ID number.  |
+|     **Type**   |     Displays the type of the member. The type can be lead, locator, or data server. The name of the active lead member is displayed in bold letters. |
+|     **CPU Usage**        |      Displays the CPU utilized by the member's host.  |
+|     **Memory Usage**       |      Displays the collective Heap and Off-Heap memory utilization of a cluster member. |
+|  **Heap Memory**       | Displays the member's utilized Heap memory versus total Heap memory. Click the down arrow in this column to view the detailed distribution of the member's Heap Memory for storage, execution, and JVM. |
+|    **Off-Heap Memory**     |     Displays the member's used Off-Heap memory and total Off-Heap memory. Click the down arrow in this column to view the detailed distribution of the member's Off-Heap memory for storage and execution.  |
+|    **Disk Space**     |     Displays the collective usage of Disk Memory used by all nodes in the cluster.  |
 
-* [Member Statistics](#memberstats)
 
-* [Member Logs](#memberlogs)
+<a id="statusofmembers"></a>
 
-The top-right side of the page displays the last updated date and time for the member.
+| Status | Description |
+|--------|--------|
+|![Running](../Images/running-status.png)|Member is running.|
+|![Stopped](../Images/stopped-status.png)|Member has stopped or is unavailable.|
 
-<a id="memberstats"></a>
-### Member Statistics
 
-![Member Stats](../Images//monitoring_memberdetails_stats.png)
+<a id="memberlogs"></a>
```

Comment:
Member Logs section is displayed in Member Details View and not in Cluster Dashboard.

Please add Member Details View section just like Cluster Dashboard (Member Details view has features like trend line graphs, various members specific stats and logs details)